Just run the following cells - no need to make any changes or even understand what you're doing!

In [1]:
from allensdk.core.brain_observatory_cache import BrainObservatoryCache
boc = BrainObservatoryCache()

This will download a file containing the data we will be using. This will likely take several minutes to run. Be patient.

In [2]:
exps = boc.get_ophys_experiments(cell_specimen_ids=[541513979], stimuli=['drifting_gratings'])
session_id = exps[0]['id']
data_set = boc.get_ophys_experiment_data(session_id)
timestamps, dff = data_set.get_dff_traces()

In [3]:
print(dff.shape)

(290, 115741)


Did you get (290, 115741)? If not, reach out for help!